.buildkite/generate-tags-pipeline.sh

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env nix-shell
```

Comment:
Don't forget to update [steps in CI configuration](https://buildkite.com/serokell/tezos-packaging-tags/steps) in order to include the run of this script before pipeline configuration uploading